Mathlib/NumberTheory/EllipticDivisibilitySequence.lean

```diff
@@ -63,9 +62,13 @@ M Ward, *Memoir on Elliptic Divisibility Sequences*
 elliptic, divisibility, sequence
 -/
 
-universe u v w
+universe u v
 
-variable {R : Type u} {S : Type v} [CommRing R] [CommRing S] (W : ℤ → R) (f : R →+* S)
+variable {R : Type u} [CommRing R]
+
+section IsEllDivSequence
+
+variable (W : ℤ → R)
 
 /-- The proposition that a sequence indexed by integers is an elliptic sequence. -/
```

Comment:
I'd rather say (here and at the top) that this is the definition of being an ES, rather than being a proposition... and the same comment applies to the several `def` below.

Reply:
These docstrings are unrelated to this PR. I wouldn't worry about them at this point, as there are downstream PRs (mostly by Junyan) that significantly modify the structure of this file.